### PR TITLE
Origin/coco fix words

### PR DESCRIPTION
**Pull request by RobertLRead**
I am creating this as a PR from Coco's work. However, I must comment:

1. The change from "Provenance" to "History" looks good.
2. This does NOT fix the 108, although it seems to try to. I tested it, and it produces an error earlier than what @chihlinc tried to do. This makes me suspect that @chihlinc did not test this---I will ask her.

I cannot find the issue for the change of "Provenance" to "History", nonetheless we did want this fixed, and this fixes it.

### CODE REVIEW

**Comment by RobertLRead** [comment]
Dear @devhawk and @K42IE,
Can one of you please review this? This fixed #108 . @chihlinc started this, and I did some touch-up work on it. It also makes the "anchor" work.

File: packages/frontend/services/azureFuncs.ts [hunk 1]
```diff
@@ -2,11 +2,18 @@
...
-    return await response.json() as { record: any, attachments?: string[], timestamp: number }[];
+    try {
+        const baseUrl = useRuntimeConfig().public.baseUrl;
+        const response = await fetch(`${baseUrl}/provenance/${deviceKey}`, {
+            method: "GET",
+        });
+        return await response.json() as { record: any, attachments?: string[], timestamp: number }[];
+    } catch (error) {
+        // probably we didn't find the key...
+        console.log(`It is likely that they key your requestd: ${deviceKey} does not exist.`);
```
Comment (K42IE) [inline packages/frontend/services/azureFuncs.ts:13]: Spelling error

**Comment by judithweng** [comment]
@RobertLRead The code looks good to me. I think you can merge this.

**Comment by RobertLRead** [comment]
Dear @K42IE , after you hear from Judith send this back to me or merge it.

**Review by K42IE — APPROVED** [review]
Tested. No errors found 👍🏻 :)

**Pull request merged**